__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -0,0 +1,294 @@
+			expect( screenshot ).toMatchImageSnapshot();
+
+			await toggleOrientation( editorPage.driver );
+			// Wait for the device to finish rotating
+			await editorPage.driver.sleep( 3000 );
+
+			// Visual test check for landscape orientation
+			screenshot = await takeScreenshot();
+			expect( screenshot ).toMatchImageSnapshot();
+
+			await toggleOrientation( editorPage.driver );
+			// Wait for the device to finish rotating
+			await editorPage.driver.sleep( 3000 );
+			await editorPage.removeBlock();
+		} );
+
+		it( 'mover buttons display in the correct positions', async () => {
+			await editorPage.addNewBlock( blockNames.columns );
+			// Wait for the modal to open
+			await editorPage.driver.sleep( 3000 );
+			// Dismiss coluns layout picker
+			await editorPage.driver
+				.elementByAccessibilityId( 'Cancel' )
+				.click();
+			// Wait for the modal to close
+			await editorPage.driver.sleep( 3000 );
+
+			// Click the block appender within the first column
+			await editorPage.driver
+				.elementByAccessibilityId( 'Column Block. Row 1' )
+				.click()
+				.click();
+
+			// Click the Columns block type button
+			const blockButton = await editorPage.findBlockButton(
+				blockNames.columns
+			);
+			if ( isAndroid() ) {
+				await blockButton.click();
+			} else {
+				await editorPage.driver.execute( 'mobile: tap', {
+					element: blockButton,
+					x: 10,
+					y: 10,
+				} );
+			}
+
+			// Wait for the modal to open
+			await editorPage.driver.sleep( 3000 );
+			// Dismiss coluns layout picker
+			await editorPage.driver
+				.elementByAccessibilityId( 'Cancel' )
+				.click();
+			await toggleOrientation( editorPage.driver );
+			// Wait for the device to finish rotating
+			await editorPage.driver.sleep( 3000 );
+
+			// Visual test check for landscape orientation
+			let screenshot = await takeScreenshot();
+			expect( screenshot ).toMatchImageSnapshot();
+
+			// Navigate upwards in block hierarchy
+			await editorPage.driver
+				.elementByAccessibilityId( 'Navigate Up' )
+				.click();
+			await editorPage.driver.sleep( 1000 );
+
+			// Visual test check for landscape orientation
+			screenshot = await takeScreenshot();
+			expect( screenshot ).toMatchImageSnapshot();
+
+			await toggleOrientation( editorPage.driver );
+			// Wait for the device to finish rotating
+			await editorPage.driver.sleep( 3000 );
+			// Navigate upwards in block hierarchy
+			await editorPage.driver
+				.elementByAccessibilityId( 'Navigate Up' )
+				.click();
+			await editorPage.driver.sleep( 1000 );
+			await editorPage.removeBlock();
+		} );
+
+		it( 'displays with correct colors with dark mode enabled', async () => {
+			if ( isAndroid() ) {
+				await editorPage.driver.executeScript( 'mobile: shell', [
+					{
+						command: 'settings put system ui_night_mode 2',
+					},
+				] );
+			} else {
+				await editorPage.driver.execute( 'mobile: setAppearance', {
+					style: 'dark',
+				} );
+			}
```

Comment:
I know we've talked about having too many helpers but I think this would be a very good case for one, I think it would be used quite a bit. What do you think?